core/util/iris/meta_search_util.py

```diff
@@ -1,26 +1,24 @@
 """
 OWASP Maryam!
-
 This program is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 any later version.
-
 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
-
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
+from math import trunc
+
 class main:
```

Comment:
Some of the cites seem misplaced
![image](https://user-images.githubusercontent.com/33282622/115680060-e4bee800-a35b-11eb-9646-3afe0f2dea31.png)

Reply:
this is iris?

Reply:
No the cites are from yahoo search I should have made it a separate comment sorry

Reply:
right. fixed.